src/test/java/com/icthh/xm/ms/entity/service/impl/EntityServiceImplIntTest.java

```diff
@@ -474,4 +475,41 @@ public void testUniqueField() {
         xmEntityRepository.saveAndFlush(entity2);
     }
 
+    @Test
+    @Transactional
+    @WithMockUser(authorities = "SUPER-ADMIN")
+    public void testSearchWithScroll() {
```

Comment:
these two test (testSearchWithScroll, testSearchScrollWithPageable1000) time was more that 7 minutes on my laptop.

Please find another way to test functionality.